demisto_sdk/tests/test_files/validate_integration_test_valid_types.py

```diff
@@ -182,7 +182,7 @@
     "systemAssociatedTypes": None,
     "associatedToAll": False,
     "unmapped": False,
-    "unsearchable": False,
+    "unsearchable": True,
```

Comment:
Some of the values are lowercased and some uppercased. Is there a reason?
If not, please let's be consistent.